web-client/src/components/WebClientMap/WebClientMap.tsx

```diff
@@ -204,7 +205,12 @@ const WebClientMap: React.FC<MapProps> = ({
         defaultZoom={zoom}
         onGoogleApiLoaded={initGoogleMapServices}
       >
-        <MyLocationControl map={googleMap || null} onClick={locateMe} />
+        <CustomMapControl
+          map={googleMap || null}
+          controlPosition={googleMapS?.ControlPosition.LEFT_TOP}
+        >
+          <MyLocationControl onClick={locateMe} />
+        </CustomMapControl>
         <OriginMarker lat={origin.lat} lng={origin.lng} isCav={isCav} />
```

Comment:
This is what seems very confusing. Why is there a control within a control?  Are they both controls?  Is one of them a control?  Please differentiate better

Reply:
Yeah this naming does need to be improved. I will clear this up along with the interface and the new components name later today!